test/color.test.js

```diff
@@ -0,0 +1,24 @@
+const host = `http://localhost:${process.env.JEST_PORT}`;
+
+const rgb2hex = (rgb) => {
+  const [, red, green, blue] = /rgb\((\d+), (\d+), (\d+)\)/.exec(rgb);
+  const parts = [red, green, blue].map(d => `0${Number(d).toString(16)}`.slice(-2));
+  return `#${parts.join('')}`;
+};
+
+test('hex codes match background color on the color documentation page', async () => {
+  await page.goto(`${host}/color/`);
+  const colors = await page.$$eval('.test-color-swatch', els => [...els].map((el) => {
+    const text = el.innerText.trim();
+    const background = window.getComputedStyle(el).backgroundColor;
```

Comment:
`getComputedStyle()` returns colors in the form of `rgb(255, 255, 255)`. I don’t believe there’s a way to get it to return hex colors, hence the conversion function above.